### PR TITLE
updated accessibiity practices

### PR DESCRIPTION
**Pull request by timwright12**

### CODE REVIEW

File: _includes/markdown/Markup.md [hunk 1]
```diff
@@ -279,9 +180,12 @@ These are not easily maintainable and can be easily lost or cause unforeseen con
...
+### Accessibility Standards
+At a minimum, all 10up projects should pass [WCAG 2.0 Level A Standards](https://www.w3.org/WAI/intro/wcag). A baseline compliance goal of Level A is due to [WCAG guideline 1.4.3](https://www.w3.org/WAI/WCAG20/quickref/#visual-audio-contrast-contrast) which requires a minimum contrast ratio on text and images, as 10up does not always control the design of a project. If a third party design presents an inherit accessibility error, that concern should be brought to the client and project team. 
+
+For design projects and projects with a global marketplace, Level AA should be the baseline goal. The accessibility level is elevated for global markets to properly comply with [EU Functional Accessibility Requirements](http://mandate376.standards.eu/standard/technical-requirements/#9), which aligns closely with WCAG 2.0 Level AA. Having direct access to the designer also allows for greater accessibility standards to be achieved.
```
Comment (tlovett1) [inline _includes/markdown/Markup.md]: What do you mean "global marketplace"?

File: _includes/markdown/Markup.md [hunk 2]
```diff
@@ -279,9 +180,12 @@ These are not easily maintainable and can be easily lost or cause unforeseen con
...
 
-At minimum, every 10up project should make use of ARIA Landmark roles, semantic headings, and alt text on images. Compliance with Section 508, or other international accessibility laws and guidelines, may be required depending on the project.
+### Accessibility Standards
+At a minimum, all 10up projects should pass [WCAG 2.0 Level A Standards](https://www.w3.org/WAI/intro/wcag). A baseline compliance goal of Level A is due to [WCAG guideline 1.4.3](https://www.w3.org/WAI/WCAG20/quickref/#visual-audio-contrast-contrast) which requires a minimum contrast ratio on text and images, as 10up does not always control the design of a project. If a third party design presents an inherit accessibility error, that concern should be brought to the client and project team.
```
Comment (tlovett1) [inline _includes/markdown/Markup.md]: The last sentence feels a bit awkward in this paragraph.

File: _includes/markdown/Markup.md (hunk 1 above)
Comment (timwright12) [reply _includes/markdown/Markup.md]: @tlovett1 Having entities outside of the US, like a company based in Europe/Asia where extra compliance is required.

File: _includes/markdown/Markup.md (hunk 2 above)
Comment (timwright12) [reply _includes/markdown/Markup.md]: Removed the sentence, I think it still worked without it.

File: _includes/markdown/Markup.md (hunk 1 above)
Comment (timwright12) [reply _includes/markdown/Markup.md]: Updated that sentence with more clarity

**Comment by timwright12** [comment]
@tlovett1 Can you confirm this is all set or if we need some more updates when you have a moment?

**Pull request merged**